semcore/d3-chart/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.17.0] - 2023-10-27
+
+### Added
+
+* `ChartLegend` component.
```

Comment:
Probably here also need to add note about `Chart` component

Reply:
it will be in next MR